finetrainers/models/cogvideox/lora.py

```diff
@@ -15,7 +15,10 @@ def load_condition_models(
     cache_dir: Optional[str] = None,
     **kwargs,
 ):
-    tokenizer = T5Tokenizer.from_pretrained(model_id, subfolder="tokenizer", revision=revision, cache_dir=cache_dir)
+    try:
+        tokenizer = T5Tokenizer.from_pretrained(model_id, subfolder="tokenizer", revision=revision, cache_dir=cache_dir)
+    except:
+        tokenizer = AutoTokenizer.from_pretrained(model_id, subfolder="tokenizer", revision=revision, cache_dir=cache_dir)
```

Comment:
Not super proud of this but we cannot do `T5Tokenizer` on the dummy T5 tokenizer checkpoint. Some sentencepiece error.